packages/components/src/Inline/Inline.stories.tsx

```diff
@@ -43,6 +43,9 @@ const meta = {
       description: 'Set the Vertical Alignment',
     },
   },
+  args: {
+    noWrap: undefined,
```

Comment:
why do you need undefined here? We can set default to false

Reply:
So it looks better in the story :D

Reply:
In the story it are two clicks then, you wrote in the docs that the default is anyway wrap so I find this unnecessary